console/console-init/ui/src/modules/iot-device/DeviceListPage.tsx

```diff
@@ -163,12 +167,26 @@ export default function DeviceListPage() {
     await setDeleteDeviceQueryVariables(variable);
   };
 
-  const onConfirmEnableSelectedDevices = () => {
-    // TODO: TO BE DONE AFTER BACKEND IS READY
+  const onConfirmEnableSelectedDevices = async (devices: IDevice[]) => {
```

Comment:
don't need to create seperate function for enabled and disabled status.
Create single function and handled both status

const onConfirmStatus=async (devices: IDevice[],status:boolean) => {
  setIsAllSelected(false);
    setSelectedDevices([]);
    const variable = {
      a: { name: projectname, namespace },
      b: devices.map(({ deviceId }) => deviceId),
      status
    };
    await setUpdateDeviceQueryVariables(variable);
}